tools/golangci-lint.sh

```diff
@@ -6,13 +6,8 @@ BUILD_ROOT="${REPO_ROOT}/build"
 BUILD_BIN="${BUILD_ROOT}/bin"
 
 NAME=golangci-lint
-RELEASE=v1.42.1
-OSX_RELEASE_256=9c0042e91218dc1dd4eb7b54e29c7331eff081b3ac3f88b0d5df89b976fcd45c
-LINUX_RELEASE_256=214b093c15863430c4b66dd39df677dab6e38fc873ded147e331740d50eea51f
-
-ARCH=amd64
-
-RELEASE_BINARY="${BUILD_BIN}/${NAME}-${RELEASE}"
+RELEASE=v1.47.3
```

Comment:
i saw you tried latest but then went back to this version, what was the problem with 1.50.0?

Reply:
@danielhochman starting with 1.48.0, Go 1.18 is required.

Reply:
I'd rather drop 1.17 then and go with 1.19. I have some other core changes planned that rely on generics anyways.